### PR TITLE
Added Netboot

### PR DESCRIPTION
**Pull request by sudo-kraken**

### CODE REVIEW

File: quadlets/netboot-xyz/netboot-xyz.container [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[Unit]
+Description=Netbook.xyz Quadlet
```
Comment (redbeardymcgee) [inline quadlets/netboot-xyz/netboot-xyz.container:2]: `Netbook` -> `Netboot`

**Review by redbeardymcgee — CHANGES_REQUESTED** [review]
I'm gonna test the request changes thing on this one, since it's just a quick typo. thanks for the quadlet, it was in my list of todo!

**Comment by sudo-kraken** [comment]
Re-review requested.

**Review by redbeardymcgee — APPROVED** [review]
Nice work

**Pull request merged**